projects/plugins/jetpack/extensions/plugins/post-publish-qr-post-panel/index.js

```diff
@@ -15,18 +23,41 @@ export const name = 'post-publish-qr-post-panel';
 
 export const settings = {
 	render: function PluginPostPublishPanelQRPost() {
-		return (
-			<PluginPostPublishPanel
-				name="post-publish-qr-post-panel"
-				title={ __( 'QR Code', 'jetpack' ) }
-				className="post-publish-qr-post-panel"
-				icon={ <QRIcon /> }
-				initialOpen={ true }
-			>
+		const panelBodyProps = {
+			name: 'post-publish-qr-post-panel',
+			title: __( 'QR Code', 'jetpack' ),
+			className: 'post-publish-qr-post-panel',
+			icon: <QRIcon />,
+			initialOpen: true,
```

Comment:
Given that this sidebar is getting to be quite crowded now that we have multiple plugins in there, I wonder if this shouldn't be closed by default?

Reply:
agree

Reply:
addressed here https://github.com/Automattic/jetpack/pull/23336/commits/97cc2e2237a2322bbd06c9fef9e03bfa065d2ffc

Reply:
Actually, after feedback from @oskosk I think it's better to keep the panel opened, Jeremy. In this way, the user will get access to the post without any UI interaction, Also,  It will be located at the bottom of the sidebar.